dev/Pager/PrototypePager/PrototypePager.Properties.cs

```diff
@@ -0,0 +1,180 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace MUXControlsTestApp
+{
+    public sealed partial class PrototypePager : Control
+    {
+        public PagerDisplayModes PagerDisplayMode
+        {
+            get { return (PagerDisplayModes)GetValue(PagerDisplayModeProperty); }
+            set { SetValue(PagerDisplayModeProperty, value); }
+        }
+        public static readonly DependencyProperty PagerDisplayModeProperty =
+            DependencyProperty.Register("PagerDisplayMode", typeof(PagerDisplayModes), typeof(PrototypePager), new PropertyMetadata(PagerDisplayModes.Auto));
+
+        /*  Universal Pager Properties
+         * 
+         */
+        public PagerTemplateSettings PagerTemplateSettings
+        {
+            get { return (PagerTemplateSettings)GetValue(PagerTemplateSettingsProperty); }
+            private set { SetValue(PagerTemplateSettingsProperty, value); }
+        }
+        public static readonly DependencyProperty PagerTemplateSettingsProperty =
+            DependencyProperty.Register("PagerTemplateSettings", typeof(PagerTemplateSettings), typeof(PrototypePager), new PropertyMetadata(default(PagerTemplateSettings)));
+
+        public int NumberOfPages
+        {
+            get { return (int)GetValue(NumberOfPagesProperty); }
+            set { SetValue(NumberOfPagesProperty, value); }
+        }
+        public static readonly DependencyProperty NumberOfPagesProperty =
+            DependencyProperty.Register("NumberOfPages", typeof(int), typeof(PrototypePager), new PropertyMetadata(5));
+
+        public int SelectedIndex
+        {
+            get { return (int)GetValue(SelectedIndexProperty); }
+            set { SetValue(SelectedIndexProperty, value); }
+        }
+        public static readonly DependencyProperty SelectedIndexProperty =
+            DependencyProperty.Register("SelectedIndex", typeof(int), typeof(PrototypePager), new PropertyMetadata(1, OnSelectedIndexChanged));
+
+        /* NumberBox & ComboBox Text Properties
+         * 
+         */
+        public string PrefixText
+        {
+            get { return (string)GetValue(PrefixTextProperty); }
+            set { SetValue(PrefixTextProperty, value); }
+        }
+        public static readonly DependencyProperty PrefixTextProperty =
+            DependencyProperty.Register("PrefixText", typeof(string), typeof(PrototypePager), new PropertyMetadata("Page"));
+
+        public string SuffixText
+        {
+            get { return (string)GetValue(SuffixTextProperty); }
+            set { SetValue(SuffixTextProperty, value); }
+        }
+        public static readonly DependencyProperty SuffixTextProperty =
+            DependencyProperty.Register("SuffixText", typeof(string), typeof(PrototypePager), new PropertyMetadata("/"));
+       
+        /* Chevron Buttons Text Properties
```

Comment:
Button does not have a TextProperty - it has Content, should this be FirstButtonContent and of type object ? That will allow text values also.

Reply:
I don't think so. The intended design is to have plain text appear next to the buttons. So these properties attach to text boxes rather than the buttons.

Reply:
This means that the text wont be clickable?

---
In reply to: [465286365](https://github.com/microsoft/microsoft-ui-xaml/pull/2977#discussion_r465286365) [](ancestors = 465286365)

Reply:
Yup that's correct. I've verified with Marissa that the text is not going to be clickable